src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs

```diff
@@ -38,6 +38,16 @@ public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags, bool support
             return dicomTag != null;
         }
 
+        public string ParseFormattedTagPath(string dicomTagPath, bool supportMultiple = false)
+        {
+            if ((!supportMultiple && dicomTagPath.Contains('.', System.StringComparison.OrdinalIgnoreCase)) || supportMultiple)
```

Comment:
>!supportMultiple [](start = 17, length = 16)

This `!supportMultiple` expression is unnecessary if you change the order of the expressions to:

```csharp
if (supportMultiple || dicomTagPath.Contains('.', System.StringComparison.OrdinalIgnoreCase))
```

Reply:
Do you want to throw on `'.'` even if it doesn't support multiple?

---
In reply to: [564811735](https://github.com/microsoft/dicom-server/pull/501#discussion_r564811735) [](ancestors = 564811735)

Reply:
I think I still need to because it means it somehow contains multiple even when we explicitly don't support them.

---
In reply to: [567146103](https://github.com/microsoft/dicom-server/pull/501#discussion_r567146103) [](ancestors = 567146103,564811735)

Reply:
Longer term, the support multiple flag won't be necessary and this entire exception can be removed


---
In reply to: [567153710](https://github.com/microsoft/dicom-server/pull/501#discussion_r567153710) [](ancestors = 567153710,567146103,564811735)

Reply:
Ok!

---
In reply to: [567154733](https://github.com/microsoft/dicom-server/pull/501#discussion_r567154733) [](ancestors = 567154733,567153710,567146103,564811735)